src/app/development/input-test/input-test.component.spec.ts

```diff
@@ -0,0 +1,191 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { InputTestComponent } from './input-test.component';
+import { Component, OnInit, ViewChild } from '@angular/core';
+
+/**
+ * Test host component to simulate parent component.
+ */
+@Component({
+    template: `
+        <app-input-test #testComp [myFirstInput]="'dada'"></app-input-test>`
+})
+class TestHostComponent1 implements OnInit {
```

Comment:
<img width="769" alt="Screen Shot 2020-06-04 at 10 52 54" src="https://user-images.githubusercontent.com/60604010/83736225-99bbb980-a651-11ea-90a2-1d324021274e.png">

Just a small linter issue, but since this is just a testing component I don't think it's very important to change

Reply:
I think we can leave it as is.